package.json

```diff
@@ -84,6 +89,10 @@
         }
       ],
       "view/item/context": [
+        {
+          "command": "tomcat.start",
+          "when": "view == tomcatServerExplorer && viewItem != runningserver"
```

Comment:
Note `viewItem != runningserver` might be a potential issue if in the future items with other `contextValue` are added into this view.